src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -53,6 +53,11 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
             return 1
         if isinstance(ex, (CLIError, CloudError, AzureException)):
             logger.error(ex.args[0])
+            try:
+                for detail in ex.args[0].error.details:
```

Comment:
is it possible `error.details` is None?

Reply:
Yes. So try-catch. error is possible to be None.